electricpy/visu.py

```diff
@@ -12,16 +12,17 @@
 import numpy as _np
 import matplotlib as _matplotlib
 import matplotlib.pyplot as _plt
+import matplotlib.colors as mcolors
```

Comment:
Please prefix with an underscore (_mcolors)

Reply:
did you just end up removing this line?

Reply:
that was an unused import